duckduckgo/iOS
# AppTP status on New Tab page

Task/Issue URL: https://app.asana.com/0/1201470200663180/1204471076973875/f
Tech Design URL:
CC:

**Description**:
This PR adds UI to the new tab page to show the status of AppTP



**Steps to test this PR**:
1. On a fresh install (one that hasn't used AppTP) nothing new should show on the new tab page
2. Connect to AppTP. There should be a banner showing AppTP is enabled and how many trackers have been blocked
3. Disable AppTP. The banner should state AppTP is disabled. Tapping the banner in either state should take you to the AppTP view.
4. With the new tab page showing, trigger some trackers and return to the app. The count should update within 30 seconds.



**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)


---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1204572500519554
